nncf/tensorflow/quantization/quantizers.py

```diff
@@ -254,6 +254,10 @@ def __init__(self, name: str, qspec: TFQuantizerSpec):
     def half_range(self):
         return self._half_range
 
+    @property
+    def signed(self) -> bool:
+        return self.signedness_to_force in (True, None)
```

Comment:
The `self.signedness_to_force` defines the sign of the symmetric quantizer in creating an instance of the `SymmetricQuantizer` and needed to create `signed_var` variable. After the `signed_var` is created, it begins to determine the sign of the symmetric quantizer and can be changed at the initialization stage.

Reply:
Done